SSA/Projects/InstCombine/tests/goals/.gitattributes

```diff
@@ -0,0 +1,3 @@
+*.lean binary linguist-generated
```

Comment:
We mark the files as `linguist-generated` and `binary`, so that git does not attempt to resolve merge conflicts in these files. marking them as `linguist-generated` ensures that github does not attempt to render these by default, which also cuts down on review noise.